### DISCUSSION TITLE
Is it possible to cut and then merge a video into the original file? Interested in mkv, mp4 and possible other extensions.

### DISCUSSION THREAD

**Discussion by flijloku** [mifi/lossless-cut#1947] [General]
Is it possible to cut and then merge a video into the original file? Interested in mkv, mp4 and possible other extensions.

**Comment by mifi** [comment]
No, but you can export to a similar file name (e.g. a suffix) and automatically delete the input file. see Settings

**Reply by flijloku** [reply]
I didn't understand the answer, to be honest. The question was whether it is possible to cut a video into several video files and then merge them into an exact copy of the original file (with a matching hash)

**Reply by mifi** [reply]
I don't understand why you would want to do that. if it's a matching hash as the original file, then it means it's the exact same file, so can just copy it?

**Reply by flijloku** [reply]
Some services do not allow you to upload videos larger than a certain size, but they do allow you to store them without paying. The source file is needed, for example, for seeding if such a need arises later.

**Reply by mifi** [reply]
sory i still don't understand what you're asking for here

**Reply by flijloku** [reply]
Archivers can do this, but they will not be video files. I'm looking for a tool that can cut videos into parts (also video files), and then merge them into the original file. I don't know how to explain it more clearly.
